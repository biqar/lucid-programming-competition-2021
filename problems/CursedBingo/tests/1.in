5
4
7
10
15
6