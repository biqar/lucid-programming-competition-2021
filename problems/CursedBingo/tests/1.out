12
42
90
210
30
